src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -118,14 +118,7 @@ private void LibraryLoaded(object sender, LibraryServices.LibraryLoadedEventArgs
         {
             // Mark all nodes as dirty so that AST for the whole graph will be
             // regenerated.
-            foreach (var node in Nodes)
-            {
-                // Mark all nodes as dirty so that AST for the whole graph will be
-                // regenerated.
-                node.ForceReExecuteOfNode = true;
-            }
-
-            OnAstUpdated();
+            MarkAllNodesDirty();
```

Comment:
The original code above sets `forceExecute` flag, but `MarkAllNodesDirty` doesn't do that internally. I'd recommend removing `MarkAllNodesDirty` and `MarkAllNodesClean` methods. Callers like this needs to have exact control over which flag to use (`ExecutionHints.Modified` or `ForceExecute`).